src/main/java/edu/gonzaga/EpidemicCard.java

```diff
@@ -2,7 +2,56 @@
 
 public class EpidemicCard extends Card {
     
-    public void triggerEpidemic(){
+    /**
+     * Constructor for the EpidemicCard class
+     * @return an EpidemicCard object
+     * 
+     * @Author Tony
+     */
+    public void triggerEmpidemic(Gameboard board) {
+        // increase the infection rate
+        increaseInfectionRate(board);
 
+        // infect
+        infect(board);
+
+        // intensify
+        intensify(board);
+    }
+
+    /**
+     * Infects cities
+     * 
+     * @param board the gameboard
+     * @return void
+     * @Author Tony
+     */
+    private void infect(Gameboard board) {
+        // infect cities
+        board.takeInfectionTurn();
+    }
+
+    /**
+     * Intensifies the game
+     * 
+     * @param board the gameboard
+     * @return void
+     * @Author Tony
+     */
+    private void intensify(Gameboard board) {
+        // shuffle the discard pile and add it to the draw pile
+        board.getPlayerDeck().putShuffledDiscardPileOnTop();
+    }
+
+    /**
+     * Increases the infection rate
+     * 
+     * @param board the gameboard
+     * @return void
+     * @Author Tony
+     */
+    private void increaseInfectionRate(Gameboard board) {
+        // increase the infection rate
+        board.changeInfectionRate();
     }
```

Comment:
@IsaacSShade Can you confirm the following logic?
1. When an epidemic card is drawn from the draw pile (Deck class), it needs to be played immediately
    * This means the epidemic card is never gonna be added to the Hand deck
1. So, when we call triggerEpidemic(), at the last step when we add the discard pile on top of the draw pile, we are essentially not dealing with the epidemic card because it has never been added to anywhere
    * This also means that we don't have to worry about removing it